src/app/launcher/mission-runtime-step/mission-runtime-step.component.html

```diff
@@ -27,22 +27,22 @@ <h2 class="card-pf-title">
             </div>
             <div class="card-pf-body">
               <div class="list-group list-view-pf" *ngFor="let runtime of runtimes; let i = index">
-                <div class="list-group-item list-view-pf-stacked list-view-pf-top-align" [ngClass]="{'disabled': runtime.disabled === true}">
+                <div class="list-group-item list-view-pf-stacked list-view-pf-top-align" [ngClass]="{'disabled': selectedMission?.id && selectedMission?.runtimes?.indexOf(runtime?.id) === -1}">
                   <div class="list-view-pf-checkbox">
                     <input name="runtime" type="radio"
-                           [disabled]="runtime.disabled === true"
+                           [disabled]="selectedMission?.id && selectedMission?.runtimes?.indexOf(runtime?.id) === -1"
                            [(ngModel)]="runtimeId"
-                           [value]="runtime.runtimeId"
+                           [value]="runtime.id"
                            (ngModelChange)="updateRuntimeSelection(runtime)">
                   </div>
                   <div class="list-view-pf-actions">
                     <div class="dropdown" dropdown>
                       <button dropdownToggle type="button" class="btn btn-default dropdown-toggle" id="{{i + 'dropdown'}}">
-                        {{runtime.version || runtime.versions[0]}}
+                        {{runtime.projectVersion || runtime.projectVersions[0]}}
```

Comment:
I'm a little confused seeing the runtime version now labeled as productVersion. Is projectVersion correct here?

FYI, there is already a projectVersion property for overall project/application, which is currently stored in the summary and selection models. That particular projectVersion number is not the same as the runtime version. 

One of these properties should probably be renamed, but need to get clarification...

Reply:
@dlabrecq Sure, we can get more clarification around this. But for now, it is followed from this https://app.swaggerhub.com/apis/fabric8-launcher/backend/2.0.0#/Booster_Catalog/get_booster_catalog_runtimes (Swagger API reference)

where there is a field projectVersion.

Reply:
Thanks @arunkumars08 , @dlabrecq needed some help here , as per swagger response for runtimes each runtime has key "missions" which is list of supported mission along with version does this version map to runtime? 

will not be using projectVersion for dropdown as it doesn't make sense , will keep versions in model as it was earlier for time being.